README.md

```diff
@@ -96,6 +96,15 @@ const orderId = await cowSdk.cowApi.sendOrder({
 
 // We can inspect the Order details in the CoW Protocol Explorer
 console.log(`https://explorer.cow.fi/rinkeby/orders/${orderId}`)
+
+// You can also override defaults params when using CowApi methods
+const orderId = await cowSdk.cowApi.sendOrder(
+  {
+    order: { ...order, ...signedOrder },
+    owner: '0x1811be0994930fe9480eaede25165608b093ad7a',
+  },
+  { chainId: 1, isDevEnvironment: false }
```

Comment:
I'm not a fan of the `isDevEnvironment` name, it even made me think in this example: `isDevEnvironment: false` is this prod? I don't know, just saying

Reply:
Mati was preparing something for this PR but yes, better if we put this in another PR.

About the naming, something that I'm thinking is that now we also have prod and staging in the subgraph. So, should we have one property and type to describe all the environments?

```typescript
type Env = 'prod' | 'staging'
```

Or specific ones for the api and subgraph?


```typescript
type CowApiEnv = 'prod' | 'barn'
type SubgraphEnv = prod' | 'staging'
```

Reply:
Yeah, I think we could have only a type with all the posibilites. Something like:

`type Env = 'prod' | 'barn' | 'staging'`

Reply:
Barn and staging are equivalent, not sure about having both on the same type, it will be confusing. If I set barn, what will be the subgraph endpoint?

Reply:
`staging` is more a standard than `barn`.
The later is only to mimic barn.cowswap.exchange, but to all effects it's the staging environment.

I'd then suggest to use only `prod` and `staging`